card/src/main/res/values/styles.xml

```diff
@@ -108,6 +108,20 @@
         <item name="drawableTint">?attr/colorOnSurface</item>
     </style>
 
+    <style name="AdyenCheckout.Card.BrandSelection.Header" parent="AdyenCheckout.TextAppearance.Primary">
```

Comment:
"AdyenCheckout.Card.BrandSelection" needs to be declared as a separate empty style to avoid build issues